packages/experiments/src/slots/SlotsPage.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import { ExampleCard, IComponentDemoPageProps, ComponentPage } from '@uifabric/example-app-base';
+
+import { SlotsExample } from './examples/Slots.Example';
+import { SlotsRootExample } from './examples/Slots.Root.Example';
+import { SlotsStackExample } from './examples/Slots.Stack.Example';
+import { SlotsIconExample } from './examples/Slots.Icon.Example';
+import { SlotsContentExample } from './examples/Slots.Content.Example';
+import { SlotsStyledExample } from './examples/Slots.Styled.Example';
+
+const SlotsExampleCode = require('!raw-loader!@uifabric/experiments/src/slots/examples/Slots.Example.tsx') as string;
+const SlotsRootExampleCode = require('!raw-loader!@uifabric/experiments/src/slots/examples/Slots.Root.Example.tsx') as string;
+const SlotsStackExampleCode = require('!raw-loader!@uifabric/experiments/src/slots/examples/Slots.Stack.Example.tsx') as string;
+const SlotsIconExampleCode = require('!raw-loader!@uifabric/experiments/src/slots/examples/Slots.Icon.Example.tsx') as string;
+const SlotsContentExampleCode = require('!raw-loader!@uifabric/experiments/src/slots/examples/Slots.Content.Example.tsx') as string;
+const SlotsStyledExampleCode = require('!raw-loader!@uifabric/experiments/src/slots/examples/Slots.Styled.Example.tsx') as string;
+
+export class SlotsPage extends React.Component<IComponentDemoPageProps, {}> {
+  public render(): JSX.Element {
+    return (
+      <ComponentPage
+        title="Slots"
+        componentName="Slots"
+        overview={
+          <div>
+            <p>Slots are a method of combining all of the various ways that subcomponents can be interacted into one prop.</p>
+          </div>
+        }
```

Comment:
Any Do's and Dont's that you'd think would be good for developers to know.

Reply:
Dos and Donts are for component pages. This is somewhat of an abuse of a component page, but it won't appear in official documentation in this format.

Reply:
The PR description is actually a much better reflection of how I see the official documentation appearing (or at least the start of it.) 😄